resources/views/product/partials/images.blade.php

```diff
@@ -4,28 +4,15 @@
             @include('rapidez::wishlist.button')
         </div>
     @endif
-    @if (count($product->images))
-        <div class="absolute inset-0 flex">
-            <div class="h-[440px] items-center justify-center rounded p-5 border sticky top-5 w-full">
-                <img
-                    class="max-h-full w-full object-contain"
-                    src="/storage/{{ config('rapidez.store') }}/resizes/400/magento/catalog/product{{ $product->images[0] }}.webp"
-                    alt="{{ $product->name }}"
-                    width="400"
-                    height="400"
-                />
-            </div>
-        </div>
-    @endif
 
-    <images v-cloak>
+    <images>
         <div class="flex-1" slot-scope="{ images, active, zoomed, toggleZoom, change }">
             <div class="sticky top-5 bg-white">
                 @include('rapidez::product.partials.gallery.slider')
-                @include('rapidez::product.partials.gallery.thumbnails')
+                @includeWhen(count($product->images) > 1, 'rapidez::product.partials.gallery.thumbnails')
```

Comment:
It's possible the configurable product does have 1 image, but a variant does have multiple. In that case the variant images won't show right?

Reply:
This is true, good point.